pkg/gofr/datasource/cassandra/cassandra_batch_test.go

```diff
@@ -15,6 +16,8 @@ func Test_BatchQuery(t *testing.T) {
 
 	values := []interface{}{1, "Test"}
 
+	ctx := context.Background()
```

Comment:
since, this is being used only once.. is it okay to create a new variable? or instead we could directly use context.Background?

Reply:
yes will refactor